make/source.mk

```diff
@@ -80,6 +80,7 @@ COMMON_SRC = \
             pg/sdcard.c \
             pg/vcd.c \
             pg/usb.c \
+            pg/sdio.c \
```

Comment:
Now that we have a parameter group for SDIO, this should go into the make feature `SDIO`, so that it only gets built (and the parameter group created) for targets that use SDIO - the granularity for conditional compilation is 'target has parameter group x or doesn't'. Sorry for not being clear about this.

Reply:
sure no problem;)